client/src/pages/Maintenance/Maintenance.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../scssStyles/mixins.scss';
+
+.full-page {
+  width: 100000vh;
+  height: 100000vh;
+
+  background: linear-gradient(
+      180deg,
+      rgba(255, 255, 255, 0) 0%,
+      rgba(221, 204, 231, 0.21875) 38.02%,
+      #621790 100%
+    ),
+    #935ebf;
+}
+
+.box {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  height: fit-content;
+  width: fit-content;
+
+  display: flex;
+  margin: auto;
+  flex-direction: column;
+  padding-top: 0;
+
+  justify-content: center;
+  align-items: center;
+}
+
+.gear-icon {
+  width: 150px;
+  height: 150px;
+  margin-bottom: 30px;
+
+  // colouring image white
+  filter: invert(100%) sepia(100%) saturate(0%) hue-rotate(160deg) brightness(105%) contrast(101%);
+
+  @include devices(mobile) {
+    width: 75px;
+    height: 75px;
+  }
+}
+
+// spinning animation
+@media (prefers-reduced-motion) {
```

Comment:
If it's reduced motion, shouldn't it not spin?
I think it can spin normally, just remove this media tag